backend/fms_core/services/utils.py

```diff
@@ -0,0 +1,19 @@
+def fit_string_with_ellipses(string: str, max_length: int, ellipses = "...") -> str:
```

Comment:
i would suggest ellipse instead of ellipses in function name and param (it sound like a list of ellipse while there is a single one). Max_length leave the impression there is a variability. I would use fitting_length or forced_length.